src/views/Browser.vue

```diff
@@ -118,6 +118,41 @@ const MAX_URIS_IN_CLAUSE = 200;
 const MAX_TERMS_PER_CATEGORY = 200;
 const MAX_PLATFORMS = 200;
 
+const PUBLICATION_FIELDS = {
+  "name": "name",  
+  "abstractText": "abstract text",
+  "authorList": "author lists",
+  "chemicals.name": "chemicals name",
+  "chemicals.registryNumber": "chemicals registry number",
+  "fullTextUri": "full text URI",
+  "keywords.term": "keywords",
+  "meshTerms.term": "mesh terms",
+  "pubAccession.accession": "accession",
+  "title": "title"
+};
+
+const HIGHLIGHT_LABELS = {
+  "shortName": "short name", 
+  "bioAssays.name": "sample name", 
+  "bioAssays.description": "sample description", 
+  "bioAssays.accession.accession": "sample accession",
+  "bioAssays.sampleUsed.name": "sample name", 
+  "bioAssays.sampleUsed.characteristics.value": "sample annotation",
+  "bioAssays.sampleUsed.characteristics.valueUri": "sample annotation URI", 
+  "characteristics.value": "annotation", 
+  "characteristics.valueUri": "annotation URI",
+  "experimentalDesign.name": "experimental design name", 
+  "experimentalDesign.description": "experimental design description", 
+  "experimentalDesign.experimentalFactors.name": "experimental factors name",
+  "experimentalDesign.experimentalFactors.description": "experimental factors description",
+  "experimentalDesign.experimentalFactors.category.categoryUri": "experimental factors category URI",
+  "experimentalDesign.experimentalFactors.category.category": "experimental factors category",
+  "experimentalDesign.experimentalFactors.factorValues.characteristics.value": "experimental factors annotation",
+  "experimentalDesign.experimentalFactors.factorValues.characteristics.valueUri": "experimental factors annotation URI",
+  ...Object.fromEntries(Object.entries(PUBLICATION_FIELDS).map(([k, v]) => ["primaryPublication." + k, "primary publication "+v])),
+  ...Object.fromEntries(Object.entries(PUBLICATION_FIELDS).map(([k, v]) => ["otherRelevantPublications." + k,"other publication "+ v]))
+}
+
```

Comment:
Can you relocate these strings under `src/config/gemma.js`?

Reply:
Added in commit e87e2d91019c10027bce31b7e505fadfe37ac34e